<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="background" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1381</width>
      <height>768</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">BACKGROUND-COLOR:rgb(145, 0, 0)</string>
    </property>
    <widget class="QTextEdit" name="Cemasys">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:WHITE;
border: none;
background-color: transparent;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#ffffff;&quot;&gt;C&lt;/span&gt;&lt;span style=&quot; font-size:28pt; font-weight:600; color:#ffffff;&quot;&gt;EMASYS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txttitle_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>-250</x>
       <y>50</y>
       <width>571</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
color:rgb(135, 0, 0);
border: none</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#ffffff;&quot;&gt;__________________________________________&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QWidget" name="frame" native="true">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>1211</width>
       <height>581</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 238);
border-radius:20px;</string>
     </property>
     <widget class="QTextEdit" name="Cemasys_9">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>641</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:WHITE;
border: none;
background-color: transparent;</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:5px; margin-bottom:5px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600; color:#7a7a7a;&quot;&gt;Add Decease's Information&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txttitle_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>1681</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color:rgb(135, 0, 0);
border: none</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#7a7a7a;&quot;&gt;____________________________________________________________________________________________________&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>240</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 11pt &quot;MS Shell Dlg 2&quot;;

color:rgb(143, 143, 143)</string>
      </property>
      <property name="text">
       <string>Date of Interment</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="Cemasys_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>451</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:WHITE;
border: none;
background-color: transparent;</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#7a7a7a;&quot;&gt;Decease's Information&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dec_lname">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>150</y>
        <width>350</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#dec_lname {
	font: 14pt &quot;MS Shell Dlg 2&quot;;
  background-color:white;
  border-radius: 20px;
  transition: box-shadow 0.3s;
}

#dec_lname:hover {
  border: 2px solid rgb(122, 122, 122);
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
  padding: 8px;
  outline: none;
}

#dec_lname:hover:focus {
  box-shadow: 0 0 8px rgb(165, 255, 240); /* Adding a glow effect */
}
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Last Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dec_fname">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>350</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#dec_fname {
	font: 14pt &quot;MS Shell Dlg 2&quot;;
  background-color:white;
  border-radius: 20px;
  transition: box-shadow 0.3s;
}

#dec_fname:hover {
  border: 2px solid rgb(122, 122, 122);
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
  padding: 8px;
  outline: none;
}

#dec_fname:hover:focus {
  box-shadow: 0 0 8px rgb(165, 255, 240); /* Adding a glow effect */
}
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>First Name</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dec_doi">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>270</y>
        <width>281</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color:white;
border-radius:20px;
color: rgb(122, 122, 122)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>240</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 11pt &quot;MS Shell Dlg 2&quot;;

color:rgb(143, 143, 143)</string>
      </property>
      <property name="text">
       <string>Date of Birth</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>240</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 11pt &quot;MS Shell Dlg 2&quot;;

color:rgb(143, 143, 143)</string>
      </property>
      <property name="text">
       <string>Date of Death</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dec_dod">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>270</y>
        <width>281</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color:white;
border-radius:20px;
color: rgb(122, 122, 122)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
     <widget class="QDateEdit" name="dec_dob">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>281</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color:white;
border-radius:20px;
color: rgb(122, 122, 122)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="dec_mname">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>150</y>
        <width>350</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#dec_mname {
	font: 14pt &quot;MS Shell Dlg 2&quot;;
  background-color:white;
  border-radius: 20px;
  transition: box-shadow 0.3s;
}

#dec_mname:hover {
  border: 2px solid rgb(122, 122, 122);
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
  padding: 8px;
  outline: none;
}

#dec_mname:hover:focus {
  box-shadow: 0 0 8px rgb(165, 255, 240); /* Adding a glow effect */
}
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Middle Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="checkbtn">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>410</y>
        <width>151</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#checkbtn {
background-color:rgb(0, 170, 0);
color: white;
font: 60 12pt &quot;Arial&quot;;
border-radius:15px;
}

#checkbtn:hover {
color:white;
padding-left: 5rem;
background-color:rgb(0, 126, 0)
}</string>
      </property>
      <property name="text">
       <string>check</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backbtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>530</y>
        <width>141</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#backbtn {
background-color:rgb(143, 143, 143);
color: black;
font: 75 15pt &quot;Arial&quot;;
border-radius:15px;
}

#backbtn:hover {
color:white;
padding-left: 5rem;
background-color: rgb(113, 113, 113)
}</string>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QComboBox" name="plot_column">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>400</y>
        <width>51</width>
        <height>40</height>
       </rect>
      </property>
      <property name="toolTipDuration">
       <number>1</number>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: white;
color:rgb(122, 122, 122);
border-radius:0px;
</string>
      </property>
      <property name="currentText">
       <string>A</string>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>10</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>C</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>D</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>E</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="plot_status">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>400</y>
        <width>261</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#plot_status {
	font: 14pt &quot;MS Shell Dlg 2&quot;;
  background-color:white;
  border-radius: 20px;
  transition: box-shadow 0.3s;
}

#plot_status:hover {
  border: 2px solid rgb(122, 122, 122);
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
  padding: 8px;
  outline: none;
}

#txtfname:hover:focus {
  box-shadow: 0 0 8px rgb(165, 255, 240); /* Adding a glow effect */
}
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Occupied</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_rec_btn">
      <property name="geometry">
       <rect>
        <x>1470</x>
        <y>520</y>
        <width>241</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#add_rec_btn {
background-color: rgb(0, 170, 0);
color:white;
font: 75 15pt &quot;Arial&quot;;
border-radius:15px;
}

#add_rec_btn:hover {
padding-left: 5rem;
background-color: rgb(0, 106, 0)
}</string>
      </property>
      <property name="text">
       <string>ADD Record</string>
      </property>
     </widget>
     <widget class="QComboBox" name="plot_name">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>400</y>
        <width>201</width>
        <height>40</height>
       </rect>
      </property>
      <property name="toolTipDuration">
       <number>1</number>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: white;
color:rgb(122, 122, 122);
border-radius:0px;
</string>
      </property>
      <property name="currentText">
       <string>Sta. Maria</string>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>10</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Sta. Maria</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sta. Monica</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San Bartolome</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San Isidro</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San Pablo</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="plot_row">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>400</y>
        <width>51</width>
        <height>40</height>
       </rect>
      </property>
      <property name="toolTipDuration">
       <number>1</number>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: white;
color:rgb(122, 122, 122);
border-radius:0px;
</string>
      </property>
      <property name="currentText">
       <string>1</string>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>10</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
     </widget>
     <widget class="QTextEdit" name="Cemasys_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:WHITE;
border: none;
background-color: transparent;</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; color:#7a7a7a;&quot;&gt;Plot Location&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1381</width>
       <height>768</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1336</width>
       <height>768</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../images/cemeterybg.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>txttitle_3</zorder>
    <zorder>frame</zorder>
    <zorder>Cemasys</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
